db/migrations/20180917141421_users_table.js

```diff
@@ -12,6 +12,11 @@ exports.up = function(knex, Promise) {
     tbl
     .string('password', 128)
     .notNullable()
+    .unique();
+
+    tbl
+    .string('role', 128)
+    .notNullable();
 
```

Comment:
Not part of MVP, but a nice starting point to implement authorization in this API.